draft-ietf-oauth-status-list.md

```diff
@@ -571,7 +573,16 @@ Resulting in the byte array and compressed/base64url-encoded status list:
 # Security Considerations {#Security}
 
 ## Correct decoding and parsing of the encoded status list
-TODO elaborate on risks of incorrect parsing/decoding leading to erroneous status data
+
+Implementers should be particularly careful for the correct parsing and decoding of the status list. Incorrect implementations might check the index on the wrong data or miscalculate the bit and byte index leading to an erroneous status of the Referenced Token. Beware, that bits are indexed (bit order) from least significant bit to most significant bit (also called "right to left") while bytes are indexed (byte order) in their natural incrementing byte order (usually written for display purpose from left to write). Endianness does not apply here because each status value fits within a single byte.
+
+Implementations shall always verify correctness using the test vectors given by this specification.
```

Comment:
Should we use stronger language here? If yes, we have probably more parts in this section, where it would make sense.
```suggestion
Implementations are RECOMMENDED to verify correctness using the test vectors given by this specification.
```